.gitlab-ci.yml

```diff
@@ -30,6 +30,27 @@ test-features:
   script:
     - cargo test --all --all-features --all-targets --locked
 
+integration-tests:
+  timeout: 5 minutes
+  image: paritytech/ci-unified:bullseye-1.70.0
+  stage: test
+  script:
+    - curl -fsSL https://deb.nodesource.com/setup_20.x | bash -
```

Comment:
We have a `.nvmrc` file, can't we simply install [nvm](https://github.com/nvm-sh/nvm) and run `nvm install` from the `chopsticks` dir? Then we don't need to install a version of Node which could get out-of-sync with our `.nvmrc` file.

Reply:
Thats a good point: [63a0d0c](https://github.com/KILTprotocol/kilt-node/pull/614/commits/63a0d0cd7ca9f852e8802b58234b9877b6bf164e)